### DISCUSSION TITLE
No sound when playing via Digital Radio

### DISCUSSION THREAD

**Discussion by TranquilHegemony** [rern/rAudio#1768] [General]
I have a Raspberry Pi4 with a HiFiBerry DAC+ HiFi hat running the latest version of rAudio but I can't get any sound when I try and play anything via DAB Radio but playing mp3 files, Web Radio and spotifyd work as expected so the problem is with DAB Radio only.  
I definitely have radio stations available and if I add one to the playlist then it appears to be playing (see attached) but I get no sound output so am I right in thinking that rAudio will take the radio stream and play this via the speakers or have I got this very wrong? 
Assuming it does work as I think does then is there any commands I can use to check further to see what is happening?
Any help gratefully received.
Thanks 
![DAB Playing](https://github.com/user-attachments/assets/39f8229f-67af-4bbe-8085-39650a9a5c69)

**Comment by klassikmann** [comment]
@TranquilHegemony 
Did you activate "FFmpeg" in the Player-Settings?

**Reply by TranquilHegemony** [reply]
Just checked and that is definitely enabled (I think it is set when you install the DAB Radio Addon as I don't recall enabling it).

**Comment by R101G** [comment]
@TranquilHegemony 
Your image shows a webradio stream, not DAB radio. I have never used DAB, but that does not look right!

**Reply by TranquilHegemony** [reply]
> @TranquilHegemony Your image shows a webradio stream, not DAB radio. I have never used DAB, but that does not look right!

That's my mistake. I was too busy pulling together the information for the post and missed the fact I had added the wrong shot. Try the following instead:
  
![DAB Radio](https://github.com/user-attachments/assets/228e2d0c-bfbe-4bbf-9526-dce74e190a44)

Thanks

**Comment by TranquilHegemony** [comment]
So looking into this a bit more. When I play an mp3 file I can see a green LED light up on the HiFiBerry DAC+ HiFi hat and this stays on while the file is playing. When I play a DAB station there is a ~5 second pause then the green LED lights up and then this stays on for ~7 seconds and then goes out again.
Checked the output of the command 'systemctl status mediamtx' and that has the following:

Sep 11 22:24:04 rAudio-Office mediamtx[17254]:     #8 0xaaaac22231ac in _start (/usr/bin/dab-rtlsdr-3+0x231ac) (Buil>
Sep 11 22:24:04 rAudio-Office mediamtx[17254]: SUMMARY: AddressSanitizer: alloc-dealloc-mismatch /build/gcc/src/gcc/>
Sep 11 22:24:04 rAudio-Office mediamtx[17254]: ==17254==HINT: if you don't care about these errors you may set ASAN_>
Sep 11 22:24:04 rAudio-Office mediamtx[17254]: ==17254==ABORTING
Sep 11 22:24:05 rAudio-Office mediamtx[310]: 2024/09/11 22:24:05 INF [RTSP] [session edca1c8b] destroyed: torn down >
Sep 11 22:24:05 rAudio-Office mediamtx[310]: 2024/09/11 22:24:05 INF [RTSP] [conn 192.168.0.240:43444] closed: termi>
Sep 11 22:24:05 rAudio-Office mediamtx[310]: 2024/09/11 22:24:05 INF [RTSP] [session 28a2e7c4] destroyed: terminated
Sep 11 22:24:05 rAudio-Office mediamtx[310]: 2024/09/11 22:24:05 INF [RTSP] [conn 127.0.0.1:58936] closed: EOF
Sep 11 22:24:05 rAudio-Office mediamtx[310]: 2024/09/11 22:24:05 INF [path 12b_c22b] runOnDemand command exited: com>
Sep 11 22:24:08 rAudio-Office mediamtx[310]: 2024/09/11 22:24:08 INF [path 12b_c22b] runOnDemand command stopped: no>

I started the DAB Radio station at ~22.37.00 to keep the times as simple to follow as possible and the above matches pretty much the times I am getting from observing the green LED going out and the session being torn down by 127.0.0.1 further supports that line of thought. My thinking is the output above is last 10 lines of the log file for the mediamtx service but what I can't work out at the minute is where the log file the information is pulled from is. Ideally I would like to see the preceding lines in that to see if the times for the earlier entries follow the same pattern and also if it offers up anything more in the way of clues so if someone could point me in the direction of that (or alternatively how to find that information) it would be appreciated. Have tried a quick Google and that's not giving me much to work with at the minute but I will keep trying.

Thanks

**Reply by R101G** [reply]
As I say, I have no experience of using the DAB dongle, but presumably some sort of initial scan is needed and this was successful, and a list of stations is saved somewhere. So the dongle is getting a signal. Trying to play and then stopping is usually a sign that the source is missing.

I am surprised the playlist is showing WWW, but you need someone who is using a DAB dongle to advise you, and I don't think there are many. It's not usually needed if you have access to the web.

**Reply by ocram76** [reply]
Hi, I also have the same problem, did you solve it, could you possibly tell me how you did it? Thank you
